src/meshapi/views/forms.py

```diff
@@ -413,6 +413,18 @@ def process_join_form(r: JoinFormRequest, request: Optional[Request] = None) ->
 
     if r.trust_me_bro:
         logging.warning(success_message)
+        if changed_info:
+            notify_string = "A new member rejected our changes to their address.\n"
+            "Please review the submission and ensure building information is accurate.\n"
+            "You might need to reach out to them and confirm their details\n"
```

Comment:
Can we emphasize that "due to a MeshDB bug" it is _extremely_ likely that human action is required in order for the database to be correct?